src/external/token/ERC20Issuance_blacklist_v1.sol

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+// Imports
+
+// Internal
+import {ERC20Issuance_v1} from "@ex/token/ERC20Issuance_v1.sol";
+import {IERC20Issuance_Blacklist_v1} 
+    from "@ex/token/interfaces/IERC20Issuance_Blacklist_v1.sol";
+
+// External
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+import {ERC20} from "@oz/token/ERC20/ERC20.sol";
+import {ERC20Capped} from "@oz/token/ERC20/extensions/ERC20Capped.sol";
+
+/**
+ * @title   ERC20 Issuance Token with Blacklist Functionality
+ *
+ * @notice  An ERC20 token implementation that extends ERC20Issuance_v1 with
+ *          blacklisting capabilities. This allows the owner to restrict specific
+ *          addresses from participating in token operations.
+ *
+ * @dev     This contract inherits from:
+ *              - IERC20Issuance_Blacklist_v1
+ *              - ERC20Issuance_v1
+ *          Key features:
+ *              - Individual address blacklisting
+ *              - Batch blacklisting operations
+ *              - Owner-controlled manager role assignment
+ *              - Blacklist manager controlled blacklist management
+ *          All blacklist operations can only be performed by the contract owner.
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer to
+ *                          our Security Policy at security.inverter.network or
+ *                          email us directly!
+ *
+ * @author  Zealynx Security
+ */
+contract ERC20Issuance_Blacklist_v1 is
+    IERC20Issuance_Blacklist_v1,
+    ERC20Issuance_v1
+{
+    //--------------------------------------------------------------------------
+    // Storage
+
+    /// @notice Mapping of blacklisted addresses
+    mapping(address => bool) private _blacklist;
+
+    /// @notice Mapping of blacklist manager addresses
+    mapping (address => bool) private _isBlacklistManager;
```

Comment:
Just seeing that we don't have any setter for this role other than the internal one called during init.

I think we need an external function here which is restricted to the owner to set a new manager. You can follow the same patterns as [here](https://github.com/InverterNetwork/contracts/blob/main/src/external/token/ERC20Issuance_v1.sol#L68)

Might be good idea to do the address zero input validation on line `84` in the internal function such that it will also be checked when the public function is called